File: src/app/classes/producto.ts
export class Producto {
    id: number;
    nombre:string;
    precio:number;
    path:string;


    constructor(id?:number)
    {
        if(id)
        this.id=id;
        else     
        this.id=0;
 
        this.precio=0;
        this.nombre="";
        this.path="";
    }

    aumentarPrecioProducto(porcentaje:number)
    {
        this.precio=this.precio+(this.precio*porcentaje);
    }

    descontarPrecioProducto(porcentaje:number)
    {
        this.precio=this.precio-(this.precio*porcentaje);
    }

}

File: BackendIndumentaria/BackendIndumentaria/Models/Producto.cs
using System;
using System.Collections.Generic;

namespace BackendIndumentaria.Models;

public partial class Producto
{
    public int Id { get; set; }

    public string? Nombre { get; set; }

    public double? Precio { get; set; }

    public string? Path { get; set; }
}
